deepchecks/nlp/checks/train_test_validation/keyword_frequency_drift.py

```diff
@@ -0,0 +1,142 @@
+        Decides which method will be used for drift calculation. Possible values are:
+        "cramer_v" for Cramer's V, "PSI" for Population Stability Index (PSI).
+    """
+
+    def __init__(self,
+                 top_n_to_show: int = 20,
+                 top_n_method: Union[str, List[str]] = 'top_diff',
+                 drift_method='cramer_v',
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.top_n_to_show = top_n_to_show
+        self.top_n_method = top_n_method
+
+        if drift_method == 'PSI':
+            self.drift_method = psi
+        elif drift_method == 'cramer_v':
+            self.drift_method = cramers_v
+        else:
+            raise DeepchecksValueError('drift_method must be one of: PSI, cramer_v')
+        self.stem_func = PorterStemmer().stem
+        self.token_pattern = r'[a-z]{2,}'
+
+        self.top_n_words = None
+        self.top_n_diffs = None
+
+    def run_logic(self, context: Context) -> CheckResult:
+        """Run check."""
+        train_dataset = context.train
+        test_dataset = context.test
+        all_data = list(train_dataset.text) + list(test_dataset.text)
+
+        vectorizer = TfidfVectorizer(input='content', strip_accents='ascii', preprocessor=self.stem_func,
+                                     token_pattern=self.token_pattern)
+        vectorizer.fit(all_data)
+        train_freqs = vectorizer.transform(train_dataset.text)
+        max_train_freqs = np.array(train_freqs.max(axis=0).todense()).reshape(-1)
+
+        test_freqs = vectorizer.transform(test_dataset.text)
+        max_test_freqs = np.array(test_freqs.max(axis=0).todense()).reshape(-1)
+        word_freq_diff = np.abs(max_train_freqs - max_test_freqs)
+
+        max_train_counts = max_train_freqs * test_dataset.n_samples
+        max_test_counts = max_test_freqs * test_dataset.n_samples
+
+        drift_score = self.drift_method(max_train_counts, max_test_counts, from_freqs=True)
+        vocab = vectorizer.get_feature_names_out()
+
+        if isinstance(self.top_n_method, List):
+            top_n_idxs = [idx for idx, word in enumerate(vocab) if word in self.top_n_method]
+        elif self.top_n_method == 'top_diff':
+            top_n_idxs = np.argsort(word_freq_diff)[-self.top_n_to_show:]
+        elif self.top_n_method == 'top_freq':
+            max_freqs = np.maximum(max_train_freqs, max_test_freqs)
+            top_n_idxs = np.argsort(max_freqs, )[-self.top_n_to_show:]
+        else:
+            raise DeepchecksValueError('top_n_method must be one of: top_diff, top_freq or a list of keywords')
+
+        top_n_words = np.take(np.array(vocab), top_n_idxs)
+        top_n_diffs = np.take(word_freq_diff, top_n_idxs)
+
+        if context.with_display:
+            train_to_show = max_train_freqs[top_n_idxs]
+            test_to_show = max_test_freqs[top_n_idxs]
+            display = word_counts_drift_plot(train_to_show, test_to_show, top_n_words)
```

Comment:
I think we should add here some text explaining what the checks does, same as we do in https://docs.deepchecks.com/stable/checks_gallery/tabular/train_test_validation/plot_train_test_feature_drift.html and other complicated checks. Explain what is a drift score (can copy the paragraph from the regular drift check), that we're showing the difference in TF-IDF and what it means.